README.md

```diff
@@ -26,13 +28,13 @@ Install GraphQL.js from npm
 With yarn:
 
 ```sh
-yarn add graphql
```

Comment:
We should probably get rid of the yarn instructions here since I removed the `yarn.lock` file in favor of `package-lock.json`.